frontend/javascripts/oxalis/view/settings/dataset_settings_view.js

```diff
@@ -114,8 +114,6 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps> {
   getReloadDataButton = (layerName: string) => {
     const tooltipText =
       "Reload the data from the server. Use this when the data on the server changed.";
-    // If the tracing contains a volume tracing, the backend can only
```

Comment:
This comment was outdated? So, can the back-end also look for data in the volume annotation? Because I think that the button is not always available for volume tracings :thinking: